crates/test-utils/src/fixtures.rs

```diff
@@ -5,16 +5,16 @@ use crate::execution::contract::KakarotEvmContract;
 use crate::sequencer::Katana;
 
 #[fixture]
-#[awt]
 pub async fn counter(#[future] katana: Katana) -> (Katana, KakarotEvmContract) {
+    let katana = katana.await;
     let eoa = katana.eoa();
     let contract = eoa.deploy_evm_contract("Counter", ()).await.expect("Failed to deploy Counter contract");
     (katana, contract)
 }
 
 #[fixture]
-#[awt]
 pub async fn erc20(#[future] katana: Katana) -> (Katana, KakarotEvmContract) {
+    let katana = katana.await;
```

Comment:
Why do we need this? (curious to learn)

Reply:
not able to see the error on CI, but if you run `./scripts/clippy.sh` you will see some errors:

>  `impl Future<Output = Katana>` doesn't implement `Send`

But `Katana` itself implements `Send` so its `Future` also should. So afaik that is a false positive and I was trying to fix that lint. but even after specifically adding `#[allow]` for that lint it was giving same warning.

Reply:
there was one more lint that didn't make any sense, i think due to this kind of false positive people in general don't like to enable them in CI

i added `clippy::nursery` just to show you guys, will remove it once you confirm its false positive.

Reply:
This is not needed, you can add the `#[awt]` macro to wait for the katana fixture to resolve.

Reply:
https://github.com/kkrt-labs/kakarot-rpc/pull/656#discussion_r1400736288

Reply:
i thought compiler is getting confused in some macro, so i was trying to remove #[awt] which manually awaits the future and did it manually, but that still didnt fix it

tldr; the issue persists with and without #[awt] macro

Reply:
but i will readd to make it what it was previously